### PR TITLE
Test PR: Test of duplicate branch of PUN Classic

### PR DESCRIPTION
**Pull request by StephenHodgson**
Overview
---
Just a test to see if the same file changes are in #3111

### CODE REVIEW

File: Assets/MixedRealityToolkit-Examples/Demos/StandardShader/Materials/ShaderBallClippingPlane.mat [hunk 1]
```diff
@@ -8,7 +8,7 @@ Material:
   m_PrefabInternal: {fileID: 0}
   m_Name: ShaderBallClippingPlane
   m_Shader: {fileID: 4800000, guid: 5bdea20278144b11916d77503ba1467a, type: 3}
-  m_ShaderKeywords: _CLIPPING_PLANE _CLIPPING_PLANE_BORDER _DIRECTIONAL_LIGHT _HOVER_LIGHT
+    m_ShaderKeywords: _CLIPPING_PLANE _CLIPPING_PLANE_BORDER _DIRECTIONAL_LIGHT _HOVER_LIGHT
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit-Examples/Demos/StandardShader/Materials/ShaderBallClippingPlane.mat]:
```suggestion
  m_ShaderKeywords: _CLIPPING_PLANE _CLIPPING_PLANE_BORDER _DIRECTIONAL_LIGHT _HOVER_LIGHT
```

File: .gitignore [hunk 2]
```diff
@@ -62,4 +62,4 @@ Assets/ThirdParty/
 Assets/ThirdParty.meta
 Assets/TextMesh Pro.meta
 Assets/TextMesh Pro/
---Version/
+--Version/
```
Comment (StephenHodgson) [inline .gitignore]:
```suggestion
--Version/

```

File: Assets/Newtonsoft.Json.dll.meta [hunk 3]
```diff
@@ -0,0 +1,97 @@
+fileFormatVersion: 2
+guid: a24d1ec71ac07b74181d4f16092f1a4d
```
Comment (StephenHodgson) [inline Assets/Newtonsoft.Json.dll.meta]: @mr0ng is there any way we can use the built in json serializer for our needs?

File: Assets/OngSharing/Singleton.cs [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+using UnityEngine;
+
+/// <summary>
+/// Be aware this will not prevent a non singleton constructor
+///   such as `T myT = new T();`
+/// To prevent that, add `protected T () {}` to your singleton class.
+/// 
+/// As a note, this is made as MonoBehaviour because we need Coroutines.
+/// </summary>
+public class Singleton<T> : MonoBehaviour where T : MonoBehaviour
```
Comment (StephenHodgson) [inline Assets/OngSharing/Singleton.cs]: @mr0ng let's also get rid of this and replace whatever uses it with a `IMixedRealityNetworkingSystem`

**Comment by davidkline-ms** [comment]
Closing per "Just a test to see if "

**Pull request closed without merging**